.github/actions/nf-test-action/action.yml

```diff
@@ -33,7 +35,7 @@ runs:
     - name: Install nf-test
       uses: nf-core/setup-nf-test@v1
       with:
-        version: "0.9.2"
+        version: "${{ env.NFTEST_VERSION }}"
```

Comment:
Yeah that's what the renovate regex had by default. We could probably get away with VER

Reply:
+1 for `NFT_VERSION`

Reply:
I'd say we use `*_VER` because that's the default env name than NXF recognize, and we used NFT_VER to be coherent.
I'd rather NFT_VER, but I'm fine with whatever as long as use the same everywhere

Reply:
ah, sorry, meant `NFT_VER`